### ISSUE TITLE
selector depth check bug

### ISSUE THREAD

**Issue by Coobaha** [rossPatton/stylint#101]
I tested source with "table indentation"
```
.a
  display:                                              block;
  .b
     display:                                           block;
```

result is:
```
Warning:  selector depth greater than 3:
Line: 17: display                         block
```
I am happy that warning is raised :) because this code is hard to maintain and produces a lot of diff, when line length is increased/decreased. But I think this should be in another lint option, like innerWhitespace

**Comment by rossPatton** [comment]
Aha! Sure.

**Comment by rossPatton** [comment]
so, i've pushed 0.9.9 which would resolve this issue, lemme know if you still see any problems

**Comment by Coobaha** [comment]
yeap, no warning now, :+1: 
hope to see option to lint this case in the future :)

**Comment by rossPatton** [comment]
Yep, prolly add it for 1.0
